content/help/faq/distributed-tracing/external-zipkin.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Can Istio send tracing information to an external ZipKin instance?
-weight: 110
+title: Can Istio send tracing information to an external Zipkin (or Zipkin-compatible) instance?
```

Comment:
"Zipkin compatible backend" or system should be fine

there's a list of things here https://zipkin.io/pages/extensions_choices.html

Reply:
done.